unit_tests/Common.d

```diff
@@ -265,17 +265,38 @@ void assertRunsDStep(
     import std.path : baseName;
     import std.format : format;
     import clang.Util : namedTempDir;
-    import std.file : readText;
+    import std.file : readText, mkdirRecurse;
+
+    string[] actualPaths;
 
-    assertFileExists(expectedPath, file, line);
-    assertFileExists(actualPath, file, line);
+    foreach (Tuple!(string, string) filesPath; filesPaths)
+    {
+        assertFileExists(filesPath[0], file, line);
+        assertFileExists(filesPath[1], file, line);
+        actualPaths ~= filesPath[1];
+    }
 
     string outputDir = namedTempDir("dstepUnitTest");
-    string outputPath = buildPath(outputDir, baseName(expectedPath));
+
+    string[] outputPaths;
```

Comment:
One for each file.
An output path is complete location of a file which is checked for the
test.